### ISSUE TITLE
Add support for bracketed paste mode

### ISSUE THREAD

**Issue by siteshwar** [att/ast#535]
[Bracketed paste mode](http://www.xfree86.org/current/ctlseqs.html#Bracketed%20Paste%20Mode) allows to differentiate between typed-in and pasted text. We should avoid directly executing copy-pasted text if text contains newlines. Bash [already supports](https://lists.gnu.org/archive/html/bug-bash/2014-10/msg00211.html) this feature through `enable-bracketed-paste` binding.

**Comment by krader1961** [comment]
Fish shell also supports bracketed paste and manages to do so without any C/C++ code. It does it entirely via fish script and user space key bindings: https://github.com/fish-shell/fish-shell/blob/master/share/functions/__fish_shared_key_bindings.fish#L110-L148

**Comment by krader1961** [comment]
Dropping the RFC label because this should definitely be implemented. It's an important safety measure for people copying/pasting example commands. Having said that I think it's pretty low priority compared to all the other things that need to be done before another ksh release can occur.
